source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,56 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``UpdateOne()``
+method. 
+
+The following example passes a query filter and an update parameter to the
+``UpdateOne()`` method, which matches a document in the ``restaurants`` collection
+by the value of its ``_id`` field and creates a new field in that document called
+``avg_rating`` with a value of *4.4*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateOne.go
+   :start-after: begin updateone
+   :end-before: end updateone
+   :emphasize-lines: 6
+   :language: go
+   :dedent:
+
+Click here <TODO: github link to file> to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the code example, the document will include the
+``avg_rating`` field with the specified value:
+
+.. code-block:: json
+   :copyable: false
+
+   // result truncated
+   { 
+      "_id" : ObjectId("5eb3d668b31de5d588f42a7a"),
+      ...
+      "name" : "Green House Cafe",
+      "restaurant_id" : "40372112",
+      "avg_rating" : 4.4
+   }
+
+Additional Information
+----------------------
+
+For more information on updating documents, specifying query filters, and
+handling potential errors, see our guide on <TODO: change a document
+fundamental page>.
+
+For more information on update operators,
+see the :manual:`MongoDB update operator reference documentation
+</reference/operator/update/#update-operators>`.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+:go-api:`UpdateOne() <mongo#Collection.UpdateOne>`
```

Comment:
This link in on the staging server goes to pkg.go.dev documentation for the `github.com/mongodb/mongo-go-driver` package, but the canonical package is `go.mongodb.org/mongo-driver`. They're technically the same, but it could be confusing if the reader doesn't realize that.

Please update to link to the docs at https://pkg.go.dev/go.mongodb.org/mongo-driver instead.

Reply:
hard coding the URL instead

Reply:
I just noticed the code documentation URL contains the `@v1.7.0` version. Is that link going to be in the published version? If so, consider removing the `@v1.7.0` so readers will be linked to the latest version.

Reply:
since our Docs are versioned, we want to link specifically to the 1.7.x api docs!